config/fgcmOutputProducts.py

```diff
@@ -0,0 +1 @@
+config.cycleNumber = 5
```

Comment:
It tripped me up that this wasn't just a file rename!

Reply:
Right, this is a completely new file which did exist in 'hsc/...' but did not in '.'. I'm not sure why it's not showing as a file rename however!